ownership-chain/precompile/asset-metadata-extender/contracts/AssetMetadataExtender.sol

```diff
@@ -70,4 +70,22 @@ interface AssetMetadataExtender {
         string calldata uloc,
         uint32 index
     ) external view returns (string memory);
+
+    /// @notice Retrieves the extension of a token's universal location by a claimer
+    /// @param _universalLocation Universal location of the token
```

Comment:
suggestion to add a "The", otherwise it looks like Location was appended to _universal. So, I suggest this:

"@param _universalLocation the provided universal location